modules/datastore/datastore.install

```diff
@@ -102,3 +102,21 @@ function datastore_schema() {
 
   return $schema;
 }
+
+/**
+ * Add 'row_number' and 'timestamp' to the dkan_post_import_job_status table.
+ */
+function datastore_update_9003(&$sandbox) {
+  $schema = \Drupal::database()->schema();
+  $table_name = 'dkan_post_import_job_status';
+
+  // Drop the current primary keys.
+  $schema->dropPrimaryKey($table_name);
+
+  $query = "
+      ALTER TABLE {$table_name}
+      ADD COLUMN row_number INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
```

Comment:
I think the convention in Druapl tables is just to use `id` for this kind of column

Reply:
I saw a table using row_num and went with that, but 'ID' seems like a better choice here. Thank you